external/fileexchange/brewermap.m

```diff
@@ -0,0 +1,528 @@
+function [map,num,typ] = brewermap(N,scheme)
```

Comment:
this one is not needed, it is already present in fieldtrip/external/brewermap. It is also recognized by and handled in `ft_hastoolbox`.